src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -337,9 +363,53 @@ class Container extends React.Component<Props, ComponentState> {
             entityNameVal: isPrebuilt ? getPrebuiltEntityName(obj.text) : prevState.isPrebuilt ? "" : prevState.entityNameVal,
         }))
     }
+
+    onDeleteEnum = (enumValue: CLM.EnumValue) => {
+        if (this.isEnumRequiredForActions(enumValue)) {
+            this.setState({
+                deleteEnumCheck: enumValue
+            })
+            return
+        }
+        else {
+            this.deleteEnum(enumValue)
+        }
+    }
+
+    deleteEnum(enumValue: CLM.EnumValue): void {
+        let index = this.state.enumValues.indexOf(enumValue)
+        if (index >= 0) {
```

Comment:
I think this method should throw an `Error` if the enum wasn't found.  Don't think we're expecting `deleteEnum` to silently fail in any cases we use it so we can safely enforce that people don't do this by accident.

Reply:
added console error